pkg/analysis/passes/signature/signature.go

```diff
@@ -61,11 +61,14 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			Version string `json:"version"`
 		} `json:"info"`
 	}
-	if err := json.Unmarshal(metadata, &data); err != nil {
+	if err := json.Unmarshal(md, &data); err != nil {
 		return nil, nil
 	}
 
-	state := getPluginSignatureState(data.ID, data.Info.Version, archiveDir, manifest)
+	state, err := getPluginSignatureState(data.ID, data.Info.Version, archiveDir, mf)
+	if err != nil {
```

Comment:
we don't want to return an error here or this will stop all the validator from running. You can add a report and return nil,nil

Reply:
cc @xnyo, everything else looks good.

Reply:
Great catch, I always forget about that! Fixed